lib/OpenQA/Worker/Engines/isotovideo.pm

```diff
@@ -119,6 +119,8 @@ sub cache_assets {
     for my $this_asset (sort keys %$assetkeys) {
         my $asset;
         my $asset_uri = trim($vars->{$this_asset});
+        # Skip UEFI_PFLASH_VARS aseet if the job won't use UEFI.
```

Comment:
Typo: it should be `asset` with two `s` rather than two `e`

Reply:
Fixed :) , my fingers were still sleepy xD